Makefile

```diff
@@ -101,6 +101,8 @@ generate: ## Run go generate
 .PHONY: generate-crds
 generate-crds: ## Generate CRDs and Go types using kubebuilder
 	go run sigs.k8s.io/controller-tools/cmd/controller-gen crd object paths=./apis/... output:crd:artifacts:config=config/crd/bases
+	@kustomize version || (code=$$?; printf "\033[0;31mError\033[0m: there was a problem with kustomize, use 'brew install kustomize' to install it\n"; exit $$code)
+	kustomize build config/crd >deploy/crds.yaml
```

Comment:
Does this tool exist automatically if a dev has kubectl installed? Or do we need to install it?

Reply:
It's technically also in kubectl, as `kubectl kustomize`, but I've always installed it separately because historically they were pretty bad at keeping it up to date. 

Lately they've been doing a better job but still:
```
❯ kubectl version --client
Client Version: v1.30.0
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
```
vs
```
❯ kustomize version
v5.4.1
```

we could just use the builtin version, maybe that way we could use it for non-dev docs...

Reply:
Yeah, if the builtin version works for this, I'd rather just use that for now.

Reply:
The one built in `kubectl` is just a stripped down version tho that defaults to `kustomize build` and doesn't have all the other commands. It should be enough for now, but something to keep in mind.